way_to_home/tests/unittests/user_profile/test_models.py

```diff
@@ -105,3 +106,21 @@ def test_update(self):
         user_profile = UserProfile.objects.get(id=self.user_profile.id)
         self.assertEqual(user_profile.first_name, new_first_name)
         self.assertEqual(user_profile.last_name, new_last_name)
+
+        is_updated = self.user_profile.update(first_name=new_first_name,
+                                              last_name=new_last_name,
+                                              telegram_id='wrong_id',
+                                              )
+        self.assertFalse(is_updated)
+
+    def test_get_by_telegram_id(self):
+        """ Provide tests for 'get_by_telegram_id' method of certain UserProfile instance """
+        expected_user = UserProfile.objects.get(telegram_id=self.user_profile.telegram_id)
+        returned_user = UserProfile.get_by_telegram_id(telegram_id=self.user_profile.telegram_id)
+        self.assertEqual(expected_user, returned_user)
```

Comment:
You may want to change it to `user_profile` instead of `user`.
Check it in other user_profile tests, please

Reply:
fixed